File: classes/parte-7-paradigmas/107-que-es-un-paradigma-y-por-que-importa/implementaciones/c/main.c
#include <stdio.h>

int main(void) {
    long n;
    if (scanf("%ld", &n) != 1) return 1;
    long suma = 0;
    for (long i = 1; i <= n; i++) suma += i;
    printf("suma=%ld\n", suma);
    return 0;
}

File: classes/parte-7-paradigmas/107-que-es-un-paradigma-y-por-que-importa/implementaciones/rust/main.rs
use std::io::Read;

fn main() {
    let mut s = String::new();
    std::io::stdin().read_to_string(&mut s).unwrap();
    let n: i64 = s.trim().parse().unwrap();
    let suma: i64 = (1..=n).sum();
    println!("suma={suma}");
}
